tests/gtest/README

```diff
@@ -0,0 +1,21 @@
+VMA Testing Environment by GTEST README
+
+This set of tests is based on Google Test C++ environment 
+https://github.com/google/googletest
+
+
+
+For testing VMA Extended API:
+
+1. Enable VMA_EXTRA_API_ENABLED in vma/vmabase.cc
```

Comment:
Did you mean: _tests/gtest/vma/vma_base.cc_ ?
Also, assuming you did - I looked in that file and and it is not clear to me what you mean by _enabling VMA_EXTRA_API_ENABLED_. Had it appeared _disabled_ in the file it may be clear what you mean by _enabling_ it, but it doesn't appear there at all.